lib/os.js

```diff
@@ -144,19 +143,60 @@ function endianness() {
 }
 endianness[Symbol.toPrimitive] = () => kEndianness;
 
+function countOnes(n) {
```

Comment:
Maybe add a comment here explaining what this function does? :)

Reply:
Done